src/view/QuestionsList.js

```diff
@@ -0,0 +1,49 @@
+
+import React from "react";
+
+const QuestionsList = ({questions, onCreateQuestion, onViewTags, onFilterByTag, onFilterByTitle}) => (
+    <div>
+        <h2>Questions</h2>
+        <br/> <br/> 
+        <label>Search: </label>
+            <input //value={search} onSearch={ e =>onSearch("userId",e.target.value)}
```

Comment:
I guess that the search does not work yet (also judging by the content of the FilterBy... components).